script/Spidertron-pathfinder.lua

```diff
@@ -37,7 +37,8 @@ function Spidertron_Pathfinder.request_path(units, _, destination)
     ---@type LuaSurface.request_path_param
     local request_params = {unit = units[1], units = units, goal = destination}
     if units[1].valid then
-        if request_params.unit.name == "constructron-rocket-powered" then
+        if request_params.unit.name == "constructron-rocket-powered"
+        or request_params.unit.name == "constructron-ss-space-spidertron" then
```

Comment:
I would turn this into a lookup table, which is much more readable and much faster too
``` lua
-- somewhere else
local is_flying_constructron = {
    ["constructron-rocket-powered"] = true,
    ["constructron-ss-space-spidertron"] = true,
}
```
```suggestion lua
        if is_flying_constructron[request_params.unit.name] then
```